components/mpas-ocean/src/shared/mpas_ocn_tracer_surface_restoring.F

```diff
@@ -250,7 +251,11 @@ subroutine ocn_get_surfaceSalinityData( streamManager, &
         integer :: ierr
 
         integer, parameter :: nSums = 2
-        real (kind=RKIND), dimension(nSums) :: reductions, sums
+        real (kind=RKIND), dimension(nSums) :: reductions
```

Comment:
@maltrud just curious: is there any reason you went with `reductions` as a name for these variables? Sums is more intuitive to me, but maybe I'm not familiar with the jargon?

Reply:
@alicebarthel  I wanted to make as few code changes as possible so I retained the same variable names that already existed whenever possible.  in the new code, "sums" isn't needed anymore, while "reductions" serves the same purpose in both the current and new versions.  probably should have done things differently many years ago, but that's the rationale.

thanks for your careful reviewing.

Reply:
@maltrud I totally understand the wisdom of making minimal code changes. As a newcomer to these routines, I allow myself to make minor comments as to readability to new users. Take it or leave it :-)